### PR TITLE
refactor: responsive portfolio breakdown

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

Closes https://github.com/ArkEcosystem/payvo-wallet/issues/998

### How to test

An easy way to test with different values on the charts is to hardcode an array of values (otherwise you are going to need to have wallets in different networks with money)

To do that:
1. open the fike `src/domains/dashboard/hooks/use-portfolio-breakdown.ts`
2. Replace the `portfolioItems` assignation with this snippet (see the latest screenshot for a diff example), also feel free to update the percents manually
```php

		const portfolioItems = [
			{
				amount: 373.133_657_93,
				convertedAmount: 302.238_262_923_300_06,
				label: "ARK",
				percent: 50.5,
			},
			{
				amount: 222.133_657_93,
				convertedAmount: 22.238_262_923_300_06,
				label: "BLK",
				percent: 21.5,
			},
			{
				amount: 222.133_657_93,
				convertedAmount: 22.238_262_923_300_06,
				label: "BIND",
				percent: 15.6,
			},
			{
				amount: 222.133_657_93,
				convertedAmount: 22.238_262_923_300_06,
				label: "XQR",
				percent: 12.3,
			},
		];
```

https://user-images.githubusercontent.com/17262776/155387355-12ffb4c8-dc80-4cc3-96c6-8fe10722a2dc.mov

https://user-images.githubusercontent.com/17262776/155387370-fae0ab37-780f-4d7f-962c-9952ca39a667.mov

https://user-images.githubusercontent.com/17262776/155387373-54f5a71b-57ec-4728-9550-48a692ab4680.mov

https://user-images.githubusercontent.com/17262776/155387377-d1cb38d4-6278-4a35-b7f6-5ed6beaf0636.mov

image

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [x] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Comment by dated** [comment]
image

These Skeletons need to be aligned to the right

**Comment by alfonsobries** [comment]
@dated ✅

**Review by dated — APPROVED** [review]

**Review by samharperpittam — CHANGES_REQUESTED** [review]
@alfonsobries - Not sure if it's just me but it appears on breakpoint 768, 640 and 375 the 'More Details' link stops working. Is it possible for you to check that please?

File: src/app/components/Graphs/LineGraph/LineGraph.tsx [hunk 1]
```diff
@@ -54,19 +55,76 @@ export const LineGraph: React.VFC<LineGraphProperties> = ({
...
+					data-testid="LineGraph__legend"
+					className="mt-3 flex justify-center sm:mb-1 sm:mt-0 sm:justify-start lg:justify-end"
+				>
 					{renderLegend(normalizedData)}
 				</div>
 			)}
 
-			<svg ref={reference} className="h-5 w-full" data-testid="LineGraph__svg">
+			{/* Colors used on the linechart added here to ensure purgecss doesnt remove them:
+				text-theme-success-600 text-theme-warning-600 text-theme-info-600 text-theme-danger-400 text-theme-hint-400
```
Comment (faustbrian) [inline src/app/components/Graphs/LineGraph/LineGraph.tsx:120]: You can whitelist those in the config so they don't get removed if they aren't used anywhere else.

**Comment by alfonsobries** [comment]
@samharperpittam cannot replicate the issue, the button is working for me in all screen sizes

the only issue I have is that the modal looks unaligned. This is happening because other elements in the page are larger than the screen but should be fixed once we implement the responsive version of the rest of the elements.
 
image

Can you please try again or give me more details (browser, etc)

**Comment by alfonsobries** [comment]
@samharperpittam only found an issue with a hidden tooltip container that was interfering with the button, I assume that was your issue, can you please try again?

**Comment by samharperpittam** [comment]
> 

Yeah that's fixed now. Thanks!

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**